### PR TITLE
feat: subsquid update v60

### PR DESCRIPTION
**Pull request by vovacha**
The main changes are related to `squid/src/handlers` folder and `squid/schema.graphql`.

**Changes:**
- add/update handlers
- schema changes
- refactoring (reduced intermediate structures amount, files amount, made most of the handlers self-contained - easier for support, imho)
- migrations flushed (we are going to restart the indexer anyway)

**Testing:**
- check that after triggering Events, corresponding row(s) were created in the squid DB.
- `pkg/dev-env/docker-compose.yml` base local setup, but the chain could be run on the host instead of docker for simplicity (I used a 3.1.60 standalone chain for testing).

**Next steps:**
- FireSquid migration -> typegen constants, use constants instead of hardcoding defaults for optional params.
- Add `OrgEnabled`, `OrgDisabled` handlers after the chain uses named args for those events.

**Auto-generated folders:**
- `squid/src/model`
- `squid/src/types`
- `squid/db/migrations`

### CODE REVIEW

File: pkg/squid/db/migrations/1656927980028-addWebToIdentity.js [hunk 1]
```diff
@@ -1,11 +0,0 @@
-module.exports = class addWebToIdentity1656927980028 {
-  name = 'addWebToIdentity1656927980028'
-
-  async up(db) {
```
Comment (2075) [inline pkg/squid/db/migrations/1656927980028-addWebToIdentity.js:4]: `web` existet before,
`web3name` and `discord` have been added. or is this an older migration?

File: pkg/squid/src/database/identity.ts [hunk 2]
```diff
@@ -36,7 +32,6 @@ async function upsertIdentity(store: Store, identity: string, data: IdentityUpse
 		entity.web = data.web;
```
Comment (2075) [inline pkg/squid/src/database/identity.ts:32]: web3name and discord are missing

**Comment by 2075** [comment]
please provide instructions what to review, a lot of these > 90 files are generated or should be generated and therefore a review is pointless. code like this feels like there should be a different approach to building this code, maybe through scaffolding, generating, etc. also i believe the migrations are generated, where? added some minor comments re identity.

**Review by 2075 — CHANGES_REQUESTED** [review]
undecided what to review :) looks good though.

File: pkg/squid/db/migrations/1656927980028-addWebToIdentity.js (hunk 1 above)
Comment (vovacha) [reply pkg/squid/db/migrations/1656927980028-addWebToIdentity.js:4]: I dropped all of the old migrations, and it's a new one. My branch was outdated.

**Comment by vovacha** [comment]
> undecided what to review :) looks good though.

To be reviewed: everything except for the `src/model` and `src/types` folders, which are auto-generated.

**Comment by 2075** [comment]
I have already "viewed" all the (92) files, what kind of feedback do you need?

File: pkg/squid/db/migrations/1656927980028-addWebToIdentity.js (hunk 1 above)
Comment (2075) [reply pkg/squid/db/migrations/1656927980028-addWebToIdentity.js:4]: how am I supposed to know this? 🗡️

**Comment by vovacha** [comment]
> 1. I have already "viewed" all the (92) files, what kind of feedback do you need?
> 
>     2. how to test?

1. I don't need any. It's just in case you feel you want to say something or disagree with the solution.
2. As I mentioned in the description, I tested it by triggering chain events and checking if a row appeared in the squid database. It means Squid processor and ETL work.

**Review by 2075 — APPROVED** [review]

**Pull request merged**